build/Package.props

```diff
@@ -0,0 +1,21 @@
+<Project>
+  <PropertyGroup>
+    <Authors>Bonsai</Authors>
+    <Copyright>Copyright © Bonsai Foundation CIC and Contributors 2022</Copyright>
+    <PackageProjectUrl>https://github.com/bonsai-rx/zeromq</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/bonsai-rx/zeromq.git</RepositoryUrl>
```

Comment:
This (along with `RepositoryType`) is no longer necessary, `SourceLink` should add it automatically.

(You might add a [`global.json`](https://github.com/bonsai-rx/bonsai/blob/555d3c43234ae78a2d844405c823ab2d356a0887/global.json) to ensure a modern-enough .NET SDK is used for building though as SourceLink is not default prior to .NET 8.)

Reply:
You are right that `RepositoryType` does not seem to be necessary, but the `RepositoryUrl` is not being included automatically for me on SDK 8.0.303. Without the explicit URL I get the following in the `nuspec` file:

```xml
<repository type="git" commit="9fdb7ae9648b3bcba8152f116e8d59a53d797153" />
```

After adding `RepositoryUrl` I get:

```xml
<repository type="git" url="https://github.com/bonsai-rx/zeromq.git" commit="9fdb7ae9648b3bcba8152f116e8d59a53d797153" />
```

After thinking through this, I'm not sure how `SourceLink` could be sure just about the URL just from the git repo since I'm not sure the URL is part of git data itself? In theory we could have gotten the repo from anywhere (or even from a local hard drive?).

Reply:
Ok, got it. What was missing is turning on `PublishRepositoryUrl`:

```xml
<PublishRepositoryUrl>true</PublishRepositoryUrl>
```

With this `SourceLink` will automatically populate the `nuspec` with the repo URL.